kleeanalysis/kleedir/info.py

```diff
@@ -52,70 +52,73 @@ def __parse_searcher(self, infofile, path):
-
-            match = self.__force_match(self.__re_explored, infofile.readline(), 'Info file "{}" does not contain a valid explored paths entry 3 lines after the end searcher tag', path)
-            self.explored_paths = int(match.group(1))
-
-            nextLine = infofile.readline()
-            try:
-              match = self.__force_match(self.__re_constructs_per_query,
-                  nextLine,
-                  'Info file "{}" does not contain a valid avg. constructs per query entry 4 lines after the end searcher tag',
-                  path)
-              self.constructs_per_query = int(match.group(1))
-              nextLine = infofile.readline() # Get the next line for the next __force_match
-            except InputError:
-              # This line might not be emitted if there are no queries
-              self.constructs_per_query = 0
-
-            match = self.__force_match(self.__re_queries, nextLine, 'Info file "{}" does not contain a valid total queries entry 5 lines after the end searcher tag', path)
-            self.queries = int(match.group(1))
-
-            match = self.__force_match(self.__re_queries_sat, infofile.readline(), 'Info file "{}" does not contain a valid valid queries entry 6 lines after the end searcher tag', path)
-            self.satisfiable_queries = int(match.group(1))
-
-            match = self.__force_match(self.__re_queries_nsat, infofile.readline(), 'Info file "{}" does not contain a valid invalid queries entry 7 lines after the end searcher tag', path)
-            self.unsatisfiable_queries = int(match.group(1))
-
-            match = self.__force_match(self.__re_queries_cex, infofile.readline(), 'Info file "{}" does not contain a valid query cex entry 8 lines after the end searcher tag', path)
-            self.cex = int(match.group(1))
-
-            line = infofile.readline().rstrip()
-            if len(line) > 0:
-                raise InputError('Info file "{}" does not contain an empty line 9 lines after the end searcher tag'.format(path))
-
-            match = self.__force_match(self.__re_instructions, infofile.readline(), 'Info file "{}" does not contain a valid total instruction entry 10 lines after the end searcher tag', path)
-            self.instructions = int(match.group(1))
-
-            match = self.__force_match(self.__re_paths, infofile.readline(), 'Info file "{}" does not contain a valid a completed paths entry 11 lines after the end searcher tag', path)
-            self.completed_paths = int(match.group(1))
-
-            match = self.__force_match(self.__re_tests, infofile.readline(), 'Info file "{}" does not contain a valid a generated tests entry 12 lines after the end searcher tag', path)
-            self.tests = int(match.group(1))
-
-            line = infofile.readline()
-            if len(line) > 0:
-                raise InputError('Info file "{}" did not end as expected.'.format(path))
+        try:
```

Comment:
These changes aren't necessary. The `Info` constructor is called in a `try` `except` block in `kleedir.py`